docs/reference/high-availability/cluster-design.asciidoc

```diff
@@ -10,15 +10,12 @@ There is a limit to how small a resilient cluster can be. All {es} clusters
 require:
 
 * One <<modules-discovery-quorums,elected master node>> node
+* At least one node for each <<modules-node,role>>.
 * At least one copy of every <<scalability,shard>>.
 
-We also recommend adding a new node to the cluster for each
-<<modules-node,role>>.
+A resilient cluster requires redundancy for every required cluster component.
+This means a resilient cluster must have:
 
-A resilient cluster requires redundancy for every required cluster component,
-except the elected master node. For resilient clusters, we recommend:
-
-* One elected master node
```

Comment:
This is true, but it's not something the user needs to worry about. As long as you have at least three master-eligible nodes Elasticsearch will look after this point automatically.